libs/wingc/src/parser.rs

```diff
@@ -731,12 +731,57 @@ impl<'s> Parser<'s> {
 			return module;
 		}
 
-		if module_name.name.starts_with("\"") && module_name.name.ends_with("\"") {
+		if module_symbol.name.starts_with("\"") && module_symbol.name.ends_with("\"") {
+			// we need to inspect the npm dependency to figure out if it's a JSII library or a Wing library
+			// first, find where the package.json is located
+			let module_name = module_symbol.name[1..module_symbol.name.len() - 1].to_string();
+			let source_dir = Path::new(&self.source_name).parent().unwrap().to_str().unwrap();
+			let module_dir =
+				wingii::util::package_json::find_dependency_directory(&module_name, &source_dir).ok_or_else(|| {
```

Comment:
Change `find_dependency_directory` to accept a Path instead of a String perhaps?